cartography/intel/gcp/apigateway.py

```diff
@@ -279,6 +279,7 @@ def load_apis_tx(tx: neo4j.Transaction, apis: List[Dict], project_id: str, gcp_u
     SET
         api.name = ap.name,
         api.createTime = ap.createTime,
+        api.location = {location},
```

Comment:
location values should be read from `apis` object. If the object has `location` or `region` related field, then we should use that.